src/docker/orb.yml

```diff
@@ -142,17 +197,29 @@ commands:
               echo "DOCKER_PASSWORD is not set, will not be able to connect to Docker Hub" >&2
               exit 1
             fi
-      - run: docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+      - run: docker login -u ${DOCKER_USERNAME} -p ${DOCKER_PASSWORD} ${DOCKER_SERVER:-}
   load_image:
     description: |
       Loads the docker image that has been built from the workspace
+    parameters:
+      docker_source_namespace:
```

Comment:
we need to use another namespace that `ledgerhq`?

Reply:
Yep we are looking at a different registry https://ledgerhq.atlassian.net/browse/PE-65 which is not called ledgerhq it seems